common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -75,6 +76,11 @@ def student_view(self, context):
 
         return fragment
 
+    def get_unit_display_name(self):  # pylint: disable=missing-docstring
```

Comment:
I had tried using only `self.display_name_with_default`, but got some python test failures "time out loading the display_name_with_default. I cleaned up this morning again, it turned out that test is subtle.